site/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I don't think we need this file, because we have a `LICENSE` at the root of this repo.